content/guides/references/troubleshooting.md

```diff
@@ -443,6 +443,13 @@ environment variable below during `cypress open` or `cypress run`.
 CYPRESS_NO_COMMAND_LOG=1 cypress run
 ```
 
+Disabling the Command Log can make tests faster as there is no time spent
+displaying the command log and updating it as tests run. The downside is,
+when there is a failure you cannot see what command failed.
+
+**Note:** In headless mode screenshots and videos will not include the Command
```

Comment:
> In headless mode screenshots and videos will not include the Command Log.

Really it's in any mode: open , run headed, or run headless. No command log will render when a spec is running.


```suggestion
**Note:** With this variable set, screenshots and videos will not include the Command Log
```